wcag-url-generator.js

```diff
@@ -0,0 +1,40 @@
+// https://github.com/w3c/wai-wcag-quickref/blob/gh-pages/_data/wcag21.json
```

Comment:
We don't need to commit this file, do we? Are we ever going to need it again?

Reply:
Yup. Keep this file for now.

As the commits in [wcag21](https://github.com/w3c/wai-wcag-quickref/blob/gh-pages/_data/wcag21.json) changes, I would like to update the success_criteria mapping. I have an idea to automate it with webhooks, so this file can be generated on the fly. But that is an overhead at the moment.